main/webapp/modules/core/scripts/project/browsing-engine.js

```diff
@@ -234,6 +242,21 @@ BrowsingEngine.prototype.removeFacet = function(facet) {
   }
 };
 
+BrowsingEngine.prototype.hideShowFacetSplash = function(clicked) {
+  var hideBrowsingFacetSplash = JSON.parse(Refine.getPreference("ui.browsing.hideFacetSplash", false));
+  var checkbox = $('#browsing-panel-checkbox');
+
+  if(checkbox.prop('checked')) {
+    Refine.setPreference('ui.browsing.hideFacetSplash', true);
+  }
+
+  if(hideBrowsingFacetSplash === true || clicked === true) {
+    $('.browsing-panel-help').addClass('hideFacetSplash');
```

Comment:
```suggestion
    $('.facet-splash-panel').addClass('hideLeftPanelSplash');
```
So, following the other comment, your styling is shared between the 2 spash, but the name must be unique and precise. `browsing-panel-help ` could be either, no?

Reply:
Oups. I might have commented to fast… I see this: `history-panel-help`. This seams to be the original name (and it is precise).